packages/meta-harness/lib/meta_harness_common.py

```
@@ -0,0 +1,943 @@
+    main_root: Path,
+    config: dict,
+    *,
+    cand_id: str,
+    manifest: dict,
+    overlay_dir: Path,
+    overlay_files: list[str],
+) -> Path:
+    """candidates/<cand_id>/ を immutable に配置する。
+
+    既に同名の候補ディレクトリが存在する場合は `FileExistsError` を送出する
+    （immutability 原則、Sec1-1「基本設計からの変更点」参照）。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)